tests/Unit/Evolution/Systems/Cce/Test_AnalyticBoundaryDataManager.cpp

```diff
@@ -0,0 +1,197 @@
+    data_set_name = subfile_name;
+    output_news_data = data_row;
+  }
+};
+
+struct TestCallWriteNews {
+  template <typename ParallelComponent, typename... DbTags,
+            typename Metavariables, typename ArrayIndex,
+            Requires<tmpl2::flat_any_v<std::is_same_v<
+                Tags::AnalyticBoundaryDataManager, DbTags>...>> = nullptr>
+  static void apply(const db::DataBox<tmpl::list<DbTags...>>& box,
+                    Parallel::ConstGlobalCache<Metavariables>& cache,
+                    const ArrayIndex& /*array_index*/) noexcept {
+    db::get<Tags::AnalyticBoundaryDataManager>(box).write_news(
+        cache, db::get<::Tags::TimeStepId>(box).substep_time().value());
+  }
+};
+
+template <typename Metavariables>
+struct mock_observer_writer {
+  using component_being_mocked = observers::ObserverWriter<Metavariables>;
+  using replace_these_threaded_actions =
+      tmpl::list<observers::ThreadedActions::WriteSimpleData>;
+  using with_these_threaded_actions = tmpl::list<MockWriteSimpleData>;
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = size_t;
+  using const_global_cache_tags = tmpl::list<>;
+
+  using phase_dependent_action_list =
+      tmpl::list<Parallel::PhaseActions<typename Metavariables::Phase,
+                                        Metavariables::Phase::Initialization,
+                                        tmpl::list<>>>;
+};
+
+template <typename Metavariables>
+struct mock_boundary {
+  using simple_tags =
+      tmpl::list<Tags::AnalyticBoundaryDataManager, ::Tags::TimeStepId>;
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = size_t;
+  using const_global_cache_tags = tmpl::list<Tags::ObservationLMax>;
+
+  using phase_dependent_action_list = tmpl::list<Parallel::PhaseActions<
+      typename Metavariables::Phase, Metavariables::Phase::Initialization,
+      tmpl::list<ActionTesting::InitializeDataBox<simple_tags,
+                                                  db::AddComputeTags<>>>>>;
+};
+
+struct metavariables {
+  using component_list = tmpl::list<mock_observer_writer<metavariables>,
+                                    mock_boundary<metavariables>>;
+  using observed_reduction_data_tags = tmpl::list<>;
+  enum class Phase { Initialization, Exit };
+};
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.AnalyticBoundaryDataManager",
+                  "[Unit][Cce]") {
+  // set up the analytic data parameters
+  MAKE_GENERATOR(gen);
+  UniformCustomDistribution<size_t> sdist{7, 10};
+  const size_t l_max = sdist(gen);
+  UniformCustomDistribution<double> parameter_dist{0.1, 1.0};
+  const double extraction_radius = 100.0 * parameter_dist(gen);
+  const double frequency = 0.01 * parameter_dist(gen);
+
+  const double time = 2.0;
+
+  Solutions::LinearizedBondiSachs analytic_solution{
+      {parameter_dist(gen) * 0.0, parameter_dist(gen) * 0.0},
```

Comment:
huh?

Reply:
hmm... suspicious. Must be a debugging remnant ... I'll revert.